MessengerProj/src/main/res/values/strings.xml

```diff
@@ -110,27 +110,27 @@
     <!--cache view-->
     <string name="CacheSettings">Storage</string>
     <string name="KeepMedia">Keep Media</string>
-    <string name="KeepMediaInfo">Photos, videos and other files from cloud chats that you have <![CDATA[<b>not accessed</b>]]> during this period will be removed from this device to save disk space.</string>
+    <string name="KeepMediaInfo">Photos, videos and other files from cloud chats you have <![CDATA[<b>yet to access</b>]]> during this period will be removed from this device to save disk space.</string>
     <string name="KeepMediaForever">Forever</string>
     <!--passcode view-->
-    <string name="Passcode">Passcode Lock</string>
-    <string name="ChangePasscode">Change Passcode</string>
-    <string name="ChangePasscodeInfo">When you set up an additional passcode, a lock icon will appear on the chats page. Tap it to lock the app.\n\nNote: if you forget the passcode, you\'ll need to delete and reinstall the app.</string>
+    <string name="Passcode">App Lock</string>
+    <string name="ChangePasscode">Change app lock</string>
+    <string name="ChangePasscodeInfo">When you set up an app lock, a lock icon will appear atop the chat page. Tap it to lock the app.\n\nNote: Should you forget it, you\'ll need to delete and reinstall the app.</string>
     <string name="PasscodePIN">PIN</string>
-    <string name="PasscodePassword">Password</string>
-    <string name="EnterCurrentPasscode">Enter your current passcode</string>
-    <string name="EnterNewFirstPasscode">Enter a passcode</string>
-    <string name="EnterNewPasscode">Enter your new passcode</string>
-    <string name="EnterYourPasscode">Enter your passcode</string>
-    <string name="ReEnterYourPasscode">Re-enter your new passcode</string>
-    <string name="PasscodeDoNotMatch">Passcodes do not match</string>
+    <string name="PasscodePassword">Pass.</string>
```

Comment:
Is there really a need to "save space"?
I have a pretty small phone and this does not need to get chopped to fit.

We could think about an alternative version of DC without vowels ;-)
"ntr yr crrnt psscd"
"Lmt ntfctns"

As an advanced setting - it could bring (a lot) attention to DC ... ... (just dreaming)

Reply:
nc id

Reply:
sorry, it must read "nc d" then :)

Reply:
My main motivation was to avoid the different pass -word -phrase -code triopoly.
I suppose oblig. xkcd. But still, this maintains parity with all, without invoking either.